clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -121,6 +121,12 @@ New checks
   Gives warnings for tagged unions, where the number of tags is
   different from the number of data members inside the union.
 
+- New :doc:`bugprone-nondeterministic-pointer-iteration-order
```

Comment:
Please sort this in alphabetically, so one entry up

Reply:
I believe I've address this. Marking as resolved.